src/platform/packages/shared/kbn-search-connectors/lib/generate_connector_name.ts

```diff
@@ -9,6 +9,8 @@
 
 import { v4 as uuidv4 } from 'uuid';
 
+import kebabCase from 'lodash/kebabCase';
```

Comment:
For the bundle size, you may want to try `import { kebabCase } from 'lodash;' instead. That should allow some tree-shaking to reduce the bundle size, hopefully.

Reply:
We had linter rules to prevent named lodash imports at some point. We apparently no longer do, so I would assume the multi-import issue has been resolved.

But anyway - Yeah, I would switch to `import { kebabCase } from 'lodash'`, see if that fixes the bundle increase. Otherwise, it's just that we reached the limit naturally, and then just update the limits via `node scripts/build_kibana_platform_plugins.js --update-limits`

Reply:
that worked! thank you for help